src/pages/image/_id.vue

```diff
@@ -1,48 +1,143 @@
 <template>
-  <div :aria-label="$t('photo-details.aria.main')">
-    <VPhotoDetails
+  <div>
+    <a
+      v-if="shouldShowBreadcrumb"
+      class="px-6 pt-4 flex flex-row items-center bg-dark-charcoal-06 font-semibold text-dark-charcoal text-xs md:text-sr"
+      :href="breadcrumbUrl"
+      @click.prevent="$router.back()"
+    >
+      <Chevron class="-ms-2" />
+      {{ $t('image-details.back') }}
+    </a>
+
+    <figure class="w-full mb-4 pt-4 md:pt-8 px-6 bg-dark-charcoal-06">
+      <img
+        v-if="!sketchFabUid"
+        id="main-image"
+        :src="image.url"
+        :alt="image.title"
+        class="h-full max-h-[500px] sm:mx-auto rounded-t-sm"
```

Comment:
```suggestion
        class="h-full max-h-[500px] mx-auto rounded-t-sm"
```

This prefix breaks the center-alignment of the image on very small mobile screens.